### DISCUSSION TITLE
Помогите начинающему

### DISCUSSION THREAD

**Discussion by SertgPro** [community/community#33081] [Other Feature Feedback, Questions, & Ideas]
Приветствую вас!

Вероятнее всего я не тут должен публиковать подобный вопрос. Но, я даже не знаю куда мне обратиться.
Если укажите направление, где мне смогут помочь, буду весьма вам боагодарен!

дело в том, что я не понимаю, как исправить ошибку?

На локальный сервер устанавливал CMS UMI и при запуске выдает такую ошибку:

Ошибка (TypeError): Argument 1 passed to DemomarketPhpExtension::getCallbackAddressId() must be of the type array, null given, called in W:\domains\TestMB.loc\classes\system\subsystems\umiTemplaters\engine\PhpTemplateEngine.php on line 81

Перейдя по указанному направлению, открыл PHP файл, строка 81 и не понимаю, что он от меня хочет?

![Скриншот 15-09-2022 00 23 34](https://user-images.githubusercontent.com/106203485/190265337-396c2a98-2dee-4109-9ad6-7ff18b151e1a.png)

Помогите пожалуйста. На просторах интернета не могу найти решение.

**Comment by jfullstackdev** [comment]
if that is PHP / Laravel, then that's a common error in development

if that field is null and you are trying to call it, say count, that will result to error, in order to avoid this use `!empty` as `if` condition because the program will not proceed to counting it if the condition is not met, it should not be empty in the first place
